site/docs/security/redaction.md

```diff
@@ -0,0 +1,88 @@
+
+# Redaction
+
+Redaction is a security feature in Estuary that allows you to block or hash fields from a capture during write time, _before_ documents are fully captured.
+Adding redaction prevents fields or unhashed values from being written to disk or surfacing in error messages.
+
+Essentially, it is the earliest point in Estuary when you can remove fields, compared against other options like [derivations](/concepts/derivations) and customizing materialization schemas with [field selection](/guides/customize-materialization-fields/#field-selection-for-materializations).
+
+This makes it an essential tool when working with PII or other sensitive data.
+
+## Redaction Strategies
+
+Estuary lets you choose how you would like to handle redacted fields with different _redaction strategies_.
+
+Redaction strategies include:
+
+* **Block**
+
+   Removes fields entirely from documents.
+
+   Blocking fields is useful when none of your downstream systems need to access any information about the field or its value.
+
+   For example, completely removing a field containing a user SSN (which ideally should already be encrypted in the source system to begin with).
+
+* **Hash**
+
+   Replaces values with SHA-256 hashed versions of the values.
+
+   Hashing fields can be useful when you would like to include stand-in values for sensitive fields in downstream systems but don't want those systems or system users to have access to the unhashed value.
+
+   For example, hashing a user email so analysts can still compile information about a user journey without seeing the user's PII.
+
+## How to Use Redaction
+
+You can redact fields using Estuary's web application. Redaction is surfaced as part of the [capture](/concepts/captures) process.
+
+1. Start by creating a new capture or editing an existing one.
+
+2. In the **Target Collections** section of the capture configuration, find the **Bindings** table.
```

Comment:
This is interesting... this is literally a `table` in the code but I have always viewed this as a `list`.

I have not strong opinion here just noting this minor difference in thinking.